### DISCUSSION TITLE
Associate Managed Property with Vertical

### DISCUSSION THREAD

**Discussion by oliver-tbs** [microsoft-search/pnp-modern-search#3738] [Q&A] [answered]
Is it possible to associate a managed property with a vertical such that when the vertical is clicked the search results display the items that were returned because they match the property?

For example:

- I have 2 verticals: **Sites** and **Category**
- I enter the term "policy" which returns 10 SharePoint sites, listed under the **Sites** vertical
- when I click the **Category** vertical it lists the 3 sites that match a managed data property called category with value "policy"

**Comment by kasperbolarsen** [comment]
In your case I would set up two Results web parts, and ensure that they are connected to the search vertical in such a way that Results web part 1 only is shown when "Sites" is selected,  and likewise for "Category" and Results web part 2. 
As each Results web part has a unique query text you should be able achieve your goal. 
The Search box web part can connected to both Results web parts

**Comment by oliver-tbs** [comment]
Thanks! I have it mostly working but I can't get the Category tab to filter on the managed property SiteCategory. I'm trying to define something like this in the tab value field. Am I on the right track?

SiteCategory:{inputQueryText}

**Reply by kasperbolarsen** [reply, marked as answer]
I am not sure what I understand the issue,  but shouldn't your Query text in the Results web part for Sites be something like Category:{searchTerms} 
You might inspect the values in the Category managed property using the debug layout or SP Editor in order to check that they match your expectations

**Reply by oliver-tbs** [reply]
Yes! That fixed my issue. Thank you.
